roads.mss

```diff
@@ -1245,14 +1257,14 @@ come in as well.
     [type='track']      { line-width: @rdz12_track; }
     [type='pedestrian']   { line-width: @rdz12_pedestrian; }
     // bridleway, footway, steps are not shown at Z12
-    [type='path'],
-    [type='footway'][can_bicycle='yes'],
-    [type='bridleway'][can_bicycle='yes'] {
+    [type='path'] {
       line-width: @rdz12_path;
     }
-    [type='cycleway'] {
+    [type='cycleway'],
+    [type='path'][designated='yes'] {
```

Comment:
We have designated, segregated and other tags exported. Not sure but we could probably factor them together. :/